src/analysis/sensitivity.jl

```diff
@@ -434,8 +454,17 @@ function outcome_map(
 
     X_q = zeros(S + 1)
     for (j, fact_t) in enumerate(target_factors)
-        X_q .= quantile(X[:, fact_t], steps)
-        for (i, s) in enumerate(X_q[1:end-1])
+        ptype = model_spec.ptype[model_spec.fieldname .== fact_t][1]
+        if ptype == "categorical"
+            fact_idx = foi_spec.fieldname .== fact_t
+            lb = foi_spec.lower_bound[fact_idx][1]
+            ub = foi_spec.upper_bound[fact_idx][1]
+            X_q .= round.(quantile(lb:1:ub, steps)) .- 1
```

Comment:
Is specifying the step value (`:1:`) really necessary? Isn't that the default?

Reply:
I don't remember this, thanks for spotting